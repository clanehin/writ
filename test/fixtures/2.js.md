```js
function up() {
  goUp();
}
```

```js
function down() {
  goDown();
}
```
