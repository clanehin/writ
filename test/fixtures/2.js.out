function up() {
  goUp();
}
function down() {
  goDown();
}
